File: clase9/InnerClass/JAVA/Main.java
public class Main{
    public static void main(String[] args){
        //crear objeto de la clase OuterClass
        OuterClass A = new OuterClass();
        System.out.println(A.x);

        //Crear un objeto de la clase InnerClass
        OuterClass.InnerClass B = A.new InnerClass();
        System.out.println(B.y);
    }
}

File: clase9/InnerClass/PYTHON/Main.py
from OuterClass import OuterClass

def main():
    #CRea el objeto de la clase OuteClass
    A = OuterClass()
    print(A.x)

    #Crea e objeto de la clase InnerCLass
    B = A.InnerClass()
    print(B.y)

if __name__ == "__main__":
    main()